### PR TITLE
vdk-ipython: add support for %%vdkingest

### PR DESCRIPTION
**Pull request by antoniivanov**
The commit introduces a new IPython magic function, `vdkingest`, which
enables data ingestion through TOML configuration within a Jupyter
notebook. This enhancement aims to simplify and streamline the data
ingestion process by allowing users to declare their data flows in a
structured format (TOML), thereby making it more accessible and
maintainable.

This is an optional feature of the plugin. If `vdk-data-sources` is not
installed, an ImportError is raised with an informative message.

(the commits will be merged as separate commits with rebase)

### CODE REVIEW

**Review by duyguHsnHsn — APPROVED** [review]
Love it, please include an example cell to the description of the PR, maybe you can even show how it replaces the job_input.ingest... methods

**Comment by murphp15** [comment]
Yeah I need a screenshot example to understand it properly.

**Comment by antoniivanov** [comment]
> Yeah I need a screenshot example to understand it properly.

What kind of screenshot: 

Here is one: 

image

**Comment by murphp15** [comment]
thats pretty cool

**Review by murphp15 — APPROVED** [review]

File: projects/vdk-plugins/vdk-ipython/README.md [hunk 1]
```diff
@@ -66,6 +66,45 @@ select * from placeholder_todo
...
+
+[sources.yourSourceId_2]
+# repeat this for as many sources you want
+# ...
+
+# Data Destination Configuration.
+## Ingestion methods and targets are the same one as those accepted by send_object_for_ingestion
+## See https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/api/job_input.py#L183
+[destinations.yourDestinationId]
+## the only required filed is method
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-ipython/README.md:92]:
```suggestion
## the only required parameter is method
```

File: projects/vdk-plugins/vdk-ipython/README.md [hunk 2]
```diff
@@ -66,6 +66,45 @@ select * from placeholder_todo
...
+## target =
+
+[destinations.yourDestinationId_2]
+# repeat this for as many destinations you want
+# ...
+
+# Data Flows from Source to Destination
+[[flows]]
+from = "yourSourceId"
+to = "yourDestinationId"
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-ipython/README.md:104]:
```suggestion
[[flows]]
from = "yourSourceId"
to = "yourDestinationId"

[[flows]]
from = "yourSourceId_2"
to = "yourDestinationId_2"
```

**Pull request merged**